### PR TITLE
feat: clock and weather added

### PR DESCRIPTION
**Pull request by bishnoimukesh**
## Deploy link:
https://deploy-preview-3--rad-wisp-06d60d.netlify.app/

### CODE REVIEW

File: src/components/clock/Clock.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.time-display{
+    margin: 0 auto;
+    text-align: center;
+    color: #fff;
+    font-size: 7rem;
+    font-weight: bold;
+}
+
+.greet-msg{
+    color: #fff;
```
Comment (ravikumar0403) [inline src/components/clock/Clock.css:10]: you can use variable here

**Review by shashikantmaurya0007 — APPROVED** [review]
add second timer in your clock too. rest looks good :)

File: src/components/weather/Weather.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    const [location, setLocation] = useState({
+        load: false,
+        coordinates: {
+            lat: "",
+            log: ""
+        },
+    });
+    const [weatherData, setWeatherData] = useState(null);
+    const onSuccess = location => {
+        console.log("succ");
```
Comment (ravikumar0403) [inline src/components/weather/Weather.jsx]: remove console log

File: src/components/weather/Weather.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    const onError = error => {
+        console.log("err");
+        setLocation({
+            load: true,
+            error,
+        })
+    }
+    console.log(location.error);
+    useEffect(() => {
+        console.log("asd");
```
Comment (ravikumar0403) [inline src/components/weather/Weather.jsx]: remove console log

File: src/components/weather/Weather.jsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                    message: "Geolocation not supported",
+                },
+            }))
+        }
+        navigator.geolocation.getCurrentPosition(onSuccess, onError)
+    }, [])
+    useEffect(() => {
+        (async () => {
+            const { data } = await axios.get(`https://api.openweathermap.org/data/2.5/weather?lat=${location.coordinates.lat}&lon=${location.coordinates.log}&appid=542f18acb500a0930e49a23a06919a1d`)
+            console.log(data)
```
Comment (ravikumar0403) [inline src/components/weather/Weather.jsx]: here as well

**Review by ravikumar0403 — APPROVED** [review]
Code is clean and folder structure is also good

File: src/components/weather/Weather.jsx (hunk 2 above)
Comment (bishnoimukesh) [reply src/components/weather/Weather.jsx]: done

File: src/components/weather/Weather.jsx (hunk 3 above)
Comment (bishnoimukesh) [reply src/components/weather/Weather.jsx]: done

File: src/components/weather/Weather.jsx (hunk 4 above)
Comment (bishnoimukesh) [reply src/components/weather/Weather.jsx]: done

**Pull request merged**